README.md

```diff
@@ -5,7 +5,7 @@
 ## Requirements
 
 In order to use `copilot-ops`, you need to have an OpenAI account with access to the GPT-3 Codex model,
-and an API token saved as the `OPENAI_API_TOKEN` environment variable.
+and an API token saved as the `OPENAI_API_TOKEN` environment variable. You must also have golang installed to build the binary.
```

Comment:
I think we want to provide instructions on how to download prebuilt binaries in the readme and create our first v0.1 release - for now we have 4 binaries (x86/arm * linux/darwin) saved as action artifacts - I find it in [main branch CI runs](https://github.com/redhat-et/copilot-ops/actions/workflows/ci.yaml?query=branch%3Amain+is%3Asuccess+event%3Apush) and see artifacts at the bottom of [the last (currently) CI run ](https://github.com/redhat-et/copilot-ops/actions/runs/2536977223)

Reply:
sure!